### ISSUE TITLE
Ability to restart xteve

### ISSUE THREAD

**Issue by BryantS11** [xteve-project/xTeVe#175]
**Is your feature request related to a problem? Please describe.**
My problem is that I would exit a stream and the xteve would not pick it up. A stream slot would be taken for that tuner which would persist until xteve is restarted. 

**Describe the solution you'd like**
A button where a user can restart xteve. In my case i have to ssh into my device and restart the docker, which is a troublesome process if i'm not near a computer.

**Describe alternatives you've considered**
If you cant restart xteve as a whole it would be great if we could be able to restart services. 
An example is to restart the hdhr service.

**Comment by mar-mei** [comment]
This is due to the client, I know some IPTV apps that do not end the stream when you press stop.
These here e.g.: TV Streams - Smart Live TV (AppleTV)

You can see that very well if you look at your network traffic or if you use xTeVe with VLC for comparison.
xTeVe, monitors every client connection. If the stream is ended properly, the counter will count down.

A restart button would solve your problem with ghost streams, but would not be the right solution in my opinion if a client does not work properly.

**Comment by MCTyler** [comment]
Restart Button = Ctrl-Alt-Del, or just exit/close xTeVe command window...simple as that.

**Comment by BryantS11** [comment]
> Restart Button = Ctrl-Alt-Del, or just exit/close xTeVe command window...simple as that.

xTeve is running on a remote device. 
So restarting it requires for me to SSH into the device and then connect to the detached window and then close xTeve and then restart it.
Depending on where I am It becomes a hassle, so not so simple all the time

**Comment by Moonbase59** [comment]
I agree with @BryantS11. There _are_ people who use _real_ operating systems, not Windows.

Other softwares (take Jellyfin for example) have "Restart" and "Shutdown" buttons in their Web UI. **Would be nice if xTeVe had that, too.**

**Comment by MCTyler** [comment]
> I agree with @BryantS11. There _are_ people who use _real_ operating systems, not Windows.
> 
> Other softwares (take Jellyfin for example) have "Restart" and "Shutdown" buttons in their Web UI. **Would be nice if xTeVe had that, too.**

HTOP will also close the process for "real" OS.
